sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClient.java

```diff
@@ -76,8 +77,8 @@
  */
 @ServiceClient(builder = EventProcessorClientBuilder.class)
 public class EventProcessorClient {
-
     private static final long BASE_JITTER_IN_SECONDS = 2; // the initial delay jitter before starting the processor
+    private static final Duration DEFAULT_STOP_TIMEOUT = Duration.ofSeconds(10);
```

Comment:
Should we use the timeout defined in the amqpretryoptions? I wonder if 10 seconds is enough because ReactorExecutor schedules one last task to run after `AmqpRetryOptions.TryTimeout` has elapsed. [Here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorExecutor.java#L180)

The scheduler related to partition pump won't end until at least TryTimeout has elapsed.

Reply:
We use `min(tryTimeout/2, SERVER_BUSY_TIMEOUT)` as a timeout in the `ReactorExecutor`

https://github.com/Azure/azure-sdk-for-java/blob/5076a39f299a1bc7672dfb23e4802e11f04845d2/sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorProvider.java#L106-L117

It'd be really surprising to have stop that lasts for 1 minute, especially the stop method that used to return fast, so given that reactor will stop in 4 sec, 10 sec should still give everything plenty of time to finish but won't be extremely long to wait.